controllers/volumereplicationgroup_controller.go

```diff
@@ -250,6 +270,33 @@ func (v *VolumeReplicationGroupReconciler) HandlePersistentVolumes(
 	return nil
 }
 
+func (v *VolumeReplicationGroupReconciler) CreateVolumeReplicationCRsFromPVC(
+	ctx context.Context,
+	pvc corev1.PersistentVolumeClaim) error {
+	cr := &ramendrv1alpha1.VolumeReplication{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pvc.Name,
```

Comment:
We may want to name the `VolumeReplication` CR differently, but this naming scheme keeps it 1-1 and hence is more user friendly as well (to easily check an object of a different kind for the same named PVC).

Just noting the thought here for now.